guides/_posts/2014-01-5-vfl.html

```diff
@@ -300,34 +301,144 @@
-@horizontal [#flexibleButton(>= 70, <= 100))];
+@h (#flexibleButton(>= 70, <= 100)));
 
 /* Equivalent CCSS */
 #flexibleButton[width] >= 70;
 #flexibleButton[width] <= 100;
 {% endhighlight %}
 
-<img class="vfl-small-img" src="../../assets/images/vfl/completeLayout.png" width="315" height="27" alt="CompleteLayout">
+
+## VFL Selectors
+All previous example of VFL expressions use basic [selectors](/guides/selectors). But you can also use combinators in VFL expression
+like this:
 
 {% highlight css %}
-/* VFL */
-@horizontal |-24-[#find]-[#findNext]-[#findField(>=30)]-24-| in(#popup);
+@v | (.featured > article ~ .title) | in(#elm > .class .class2"_fallout")
+{% endhighlight %}
+
+<a name="vflsplats"></a>
+## VFL Splat
+When a selector in your VFL expression finds more than one element, due to [plural binding](/guides/selectors/#pluralbinding),
+if the number of element within each selectors expression are not equal, then some elements will not be constraint.
+<a href="http://codepen.io/cbchouinard/pen/vEJqbG" target="_blank">Live example.</a>
+
+In this example, the last two `article` elements will not be constrained since there's no `section` element to bind to.
+As you can see in the CCSS generated constraints, the elements are pluraly binded.
+
+{% highlight css %}
+
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<footer></footer>
+
+<style type="text/gss">
+
+  /* the last two articles will not be constraints because of plural binding */
+  @h |-(section)-(article)-(footer)-| in(::window);
+
+  /* this generates the following CCSS */
+  ::window[left] + [hgap] == section[left];
+  section[right] + [hgap] == article[left]; /* plural binding */
+  article[right] + [hgap] == footer[left]; /* plural binding */
+  footer[right] + [hgap] == ::window[right];
+
+</style>
+
+{% endhighlight %}
+
+If the number of selected elements matches, the position of the elements within each collection of selected
+elements will overlap since you're basically constraining the whole collection to the same values. <a href="http://codepen.io/cbchouinard/pen/yyodWx" target="_blank">Live example.</a>
+
+{% highlight html %}
+
+<section></section>
+<section></section>
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<footer></footer>
+
+<style type="text/gss">
+
+/* the 3 sections will overlap over each other as the 3 articles. */
+@h |-(section)-(article)-(footer)-| in(::window);
+
+</style>
+
+{% endhighlight %}
+
+Splats will iterate over the collection of selected elements and position elements
+next to each other. <a href="http://codepen.io/cbchouinard/pen/ogerrX" target="_blank">Live example.</a>
+
+{% highlight css %}
+
+<section></section>
+<section></section>
+<article></article>
+<article></article>
+<article></article>
+<div></div>
+<div></div>
+
+<style type="text/gss">
+
+/* elements are not 'pluraly' binded */
+@h |-(section)-10-...-(article)-20-...-(div)...-| in(::window);
```

Comment:
We seem to dive right into this complicated example with splats and then rush past them.

It would be good to build up to it, starting with something like this:

```
<section></section>
<section></section>
<section></section>

<style type="text/gss">
  @h |(section)...| in(::window);
</style>
```

Then, introducing gaps:

```
<section></section>
<section></section>
<section></section>

<style type="text/gss">
  @h |-(section)-...-| gap(10) outer-gap(20) in(::window);
</style>
```

After that, multiple elements and gaps wouldn't be so much of a leap.